ansible-ipi-install/roles/bootstrap/tasks/10_load_inv.yml

```diff
@@ -122,6 +122,7 @@
   copy:
     dest: "/tmp/metadata.txt"
     content: "{{ lookup('template', 'node_inv.j2') | to_nice_json  }}"
+  become: true
```

Comment:
Why is this needed?

Reply:
Basically `airflow` user runs a cli command which triggers the [script](https://github.com/mukrishn/airflow-kubernetes/blob/latest_patch/dags/baremetal_openshift/scripts/install_cluster.sh) inside the container.
Airflow user doesn't seems to write anything inside container without privilege escalation, `/tmp` is an `EmptyDir` volume mount. 

ex - Airflow command looks like this - `airflow tasks run 4.7_baremetal_jetski install "2021-05-11T14:55:51.829520+00:00" --local --pool default_pool --subdir /opt/airflow/dags/repo/dags/baremetal_openshift/dag.py`

Reply:
so still need this?

Reply:
yes, this one is needed.